frontend/src/components/Main.js

```diff
@@ -27,7 +27,7 @@ export function Main({ userState, setUserState }) {
     }
     return (
         <>
-            <ToDoLists lists={lists} selectedList={selectedList} setSelectedList={setSelectedList} />
+            <ToDoLists lists={lists} selectedList={selectedList} setSelectedList={setSelectedList} addList={addList}/>
```

Comment:
This code is not formatted

Reply:
submitting again, not sure if any change happened, format document doesn't seem to have done anything